modules/nf-core/ltrfinder/main.nf

```diff
@@ -0,0 +1,55 @@
+process LTRFINDER {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/edta:2.2.0--hdfd78af_0':
+        'biocontainers/edta:2.2.0--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("*.scn")      , emit: scn
+    tuple val(meta), path("*.gff3")     , emit: gff
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args            = task.ext.args ?: ''
+    def prefix          = task.ext.prefix ?: "${meta.id}"
+    def path_prefix     = (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) ? '\$CONDA_PREFIX' : '/usr/local'
+    """
+    $path_prefix/share/EDTA/bin/LTR_FINDER_parallel/LTR_FINDER_parallel \\
```

Comment:
This will still be problematic if the user does a local install of the tool and uses that. Normally one should assume the tool is in the `PATH`. Would it be possible to fix the bioconda recipe so that it's found? That'll fix it for the container and conda package.

Reply:
Good idea, I'll try to create a pull request for a change in the Bioconda recipe for EDTA or create a new recipe for LTR_FINDER_parallel.

Reply:
For future reference: https://github.com/bioconda/bioconda-recipes/pull/45280

Reply:
Hi @mahesh-panchal 

I have updated the bioconda package and containers. I also thought about placing this module under EDTA as EDTA_LTRFINDER but then did not do so because LTR_FINDER and LTR_FINDER_parallel both have their own GitHub repos. I think in this case, we should ideally have a separate light weight bioconda package for LTR_FINDER_parallel. I did not create an independent package to avoid additional maintenance overhead. Soon I have a meeting with the author of EDTA where we are going to discuss porting the EDTA workflow to a DSL2 workflow. I'll suggest that his Lab creates and maintains a separate bioconda package for each tool which has its own repo. For now, I hope you are okay with the EDTA bioconda package. Thank you!

Reply:
Hi @GallVp, EDTA author here. I came across this thread. I agree with you that LTR_FINDER_parallel can have its own recipe. If you can help to create a new recipe that would be great as I am not as familiar. This may also be the case for programs included in the EDTA/bin folder, including LTR_HARVEST_parallel, HelitronScanner, and TIR-Learner3.0. Please let me know if I can provide further help.